src/contracts/transfer-gateway.ts

```diff
@@ -141,6 +142,29 @@ export class TransferGateway extends Contract {
     })
   }
 
+   /**
+   * Adds a contract mapping to the DAppChain Gateway using gatway owner signature.
+   * A contract mapping associates a token contract on the DAppChain with it's counterpart on Ethereum.
+   */
+  addAuthorizedContractMappingAsync(params: {
+    foreignContract: Address
+    localContract: Address
+    gatewayOwnerSig: Uint8Array
```

Comment:
The `AddAuthorizedContractMapping` method doesn't accept a `gatewayOwnerSig`, should only send it the contract addresses. Only the gateway owner will be calling this method, and hence singing the tx.

Reply:
removed